sigs.yaml

```diff
@@ -492,23 +492,30 @@ sigs:
       establishment of conventions for writing CLI commands, POSIX compliance,
       and improving the command line tools from a developer and devops user
       experience and usability perspective.
-    charter_link:
+    charter_link: charter.md
     label: cli
     leadership:
       chairs:
+      - name: Maciej Szulik
```

Comment:
duplicated entry

Reply:
Nope, there are chairs which are responsible for organizational issue with sig and leads which are responsible for technicalities. Sean and I are both chairs, on top of that Phil and I are tech-leads, which makes me deserve double entry :1st_place_medal: